### ISSUE TITLE
Pylatex issue with 1.0.0 version: "\" are converted to "\texbacklash"

### ISSUE THREAD

**Issue by Delosari** [JelteF/PyLaTeX#115]
Hi JelteF! First of all I want to thank you for this great project. I had never been so excited for about a python library.

I am having an issue with version 1.0.0 which I did not experience before: I use pylatex to generate tables which I crop using the package preview. I attach an example I made with pylatex
[Example_TableOutput.pdf](https://github.com/JelteF/PyLaTeX/files/105547/Example_TableOutput.pdf). However, now the compilation fails with this error:

Command '[u'latexmk', u'--pdf', u'--interaction=nonstopmode', u'multirow.tex']' returned non-zero exit status 12

However, I think the issue arises from the tex file generated, which is full "\textbacklash" commands instead of just "\" symbols. (I attach the output tex file [multirow_withIssue.txt](https://github.com/JelteF/PyLaTeX/files/105545/multirow_withIssue.txt)) 

Finally, I have made an example of the code which generates this issue from your multirow example. Do you think you could take a look at it and tell me what is wrong
[pylatex_example_with_issue.txt](https://github.com/JelteF/PyLaTeX/files/105552/pylatex_example_with_issue.txt)?

Also I wanted to ask you how should we cite you if we publish a code which employs your library (if it is allowed)

Thanks

**Comment by JelteF** [comment]
The largest change in version 1.0.0 was the change to escape everything by default. This was for safety reasons, so no accidental latex code would be executed when it was meant as normal text.

I see you're using raw latex strings without unescaping them, this will turn `\` characters into `\textbackslash{}`. See this on how to resolve the issue:
https://jeltef.github.io/PyLaTeX/current/usage.html#unescaping-strings

It might also be possible to subclass some of the existing classes. You might subclass `Table` to create your own `TableStar` class. Or at least subclass `Environment` from the `base_classes` submodule. See:
https://jeltef.github.io/PyLaTeX/current/pylatex/pylatex.base_classes.containers.html#pylatex.base_classes.containers.Environment
And: https://jeltef.github.io/PyLaTeX/current/pylatex/pylatex.table.html#pylatex.table.Table

I'll close this for now, feel free to reopen if there is another issue.

PS. as for the attribution. If you copy the files you should also copy the original LICENSE file and copyright notices as specified by the MIT license included in the project: https://github.com/JelteF/PyLaTeX/blob/master/LICENSE
TL;DR version: https://tldrlegal.com/license/mit-license
